src/scripts/validation.js

```diff
@@ -51,9 +51,10 @@ const toggleButtonState = (inputList, buttonElement, settings) => {
 
 export const disableButton = (buttonElement, settings) => {
   buttonElement.classList.add(settings.inactiveButtonClass);
+  buttonElement.disabled = true;
 };
 
```

Comment:
* There is an error in the console https://snipboard.io/XS1wTa.jpg


Please, add ` if (!buttonElement) return;` on line 69, to prevent validation of the `delete form` because it doesn't have `modal__save-button` in the button, so it causes the error


```js
const setEventListeners = (formElement, settings) => {
  const inputList = Array.from(
    formElement.querySelectorAll(settings.inputSelector)
  );
  const buttonElement = formElement.querySelector(
    settings.submitButtonSelector
  );

  if (!buttonElement) return;  // here

```